sabin18/PropertyPro-lite-
# #167288127 add flag controller with queries

#### What does this PR do?
- update property  controller with queries
#### Description of Task to be completed
- import queries
- add queries to flag controller
- add queries to  flag model 
- import execute function to my model and controller 

#### How should this be manually tested?
-#### How should this be manually tested?
- clone the repository : git clone https://github.com/sabin18/PropertyPro-lite-.git
- checkout to branch Ft-flag-controller 167288127
- Run a command npm start
- after go to add flag, get flags,get one flag ,get flag on specific property
- POST | /api/v1/flags/
- GET | /api/v1/flags/:id
- GET | /api/v1/property/:id/flags

#### Any background context you want to provide?
`N/A`
#### What are the relevant pivotal tracker stories?
[#167288127](https://www.pivotaltracker.com/n/projects/2356989)
#### Screenshots (if appropriate
`N/A`
